doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -3169,7 +3342,52 @@ DFU and FOTA
 
 The issues in this section are related to :ref:`app_dfu`.
 
-.. rst-class:: v2-7-0 v2-6-2 v2-6-1 v2-6-0 v2-5-3 v2-5-2 v2-5-1 v2-5-0 v2-4-4 v2-4-3 v2-4-2 v2-4-1 v2-4-0
+.. rst-class:: v2-8-0
+
+NCSDK-30161 DFU from external flash does not work
+
+  **Affected platforms:** nRF54H20
+
+.. rst-class:: v2-8-0
+
+NCSDK-28241: Failed to execute SUIT payload fetch
```

Comment:
Seems that we can remove this issue from the list, because now sporadic issue has been replaced by "NCSDK-30161 DFU from external flash does not work"

Reply:
Okay, will do. But your previous comment is hardly something we want to say in the docs. Can you formulate something customer oriented?